<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Simcond_Dialog</class>
 <widget class="QDialog" name="Simcond_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Setting</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="NowSet">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>60</y>
     <width>194</width>
     <height>24</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>1970</year>
     <month>1</month>
     <day>1</day>
    </datetime>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Time Now:</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="IntervalSet">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>118</width>
     <height>24</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>30</minute>
     <second>0</second>
     <year>2000</year>
     <month>1</month>
     <day>1</day>
    </datetime>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Step:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>170</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="minimum">
    <number>10</number>
   </property>
   <property name="maximum">
    <number>200</number>
   </property>
   <property name="singleStep">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Speed:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Simcond_Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Simcond_Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
